opentelemetry-semantic-conventions/src/opentelemetry/semconv/_incubating/attributes/gen_ai_attributes.py

```diff
@@ -302,10 +302,14 @@ class GenAiSystemValues(Enum):
     """Anthropic."""
     COHERE = "cohere"
     """Cohere."""
-    AZ_AI_INFERENCE = "az.ai.inference"
+    AZURE_AI_INFERENCE = "azure.ai.inference"
     """Azure AI Inference."""
-    AZ_AI_OPENAI = "az.ai.openai"
+    AZURE_AI_OPENAI = "azure.ai.openai"
     """Azure OpenAI."""
+    AZ_AI_INFERENCE = "az.ai.inference"
+    """Deprecated: Replaced by azure.ai.inference."""
+    AZ_AI_OPENAI = "azure.ai.openai"
```

Comment:
See https://github.com/open-telemetry/semantic-conventions/pull/2486

Reply:
This has been fixed in a different way and I don't think semconv SIG consider this a blocker